domain-analysis/operations.py

```diff
@@ -0,0 +1,104 @@
+import arrow
+import os
+import whodap
+import validators
+import tldextract
+from tranco import Tranco
+from connectors.core.connector import get_logger, ConnectorError
+from .freq.freq import *
+
+
+logger = get_logger('domain_analysis')
+CACHE_DIR = '/tmp'
+FREQTABLE = os.path.dirname(os.path.realpath(__file__)) + '/freq/freqtable.freq'
+
+
+def analyze_domain(config, params):
+    try:
+        domain = params.get('domain')
+        fc = FreqCounter()
+        fc.load(FREQTABLE)
+        total_word_probability, average_probability = fc.probability(domain)
+        return {'Total Word Probability': total_word_probability, 'Average Probability': average_probability}
```

Comment:
any specific reason the keys are space separate words, and not camelCased json keys?

Reply:
done